Defaults KeyboardNavigation.IsTabStop of TransitioningContent to "False"

LoRdPMN opened the pull request:
Sets KeyboardNavigation.IsTabStop to false in OnApplyTemplate().

Transitioning content was getting keyboard focus via tab key on its content area.

ButchersBoy commented:
This should be set in the style, here:

https://github.com/ButchersBoy/MaterialDesignInXamlToolkit/blob/master/MaterialDesignThemes.Wpf/Themes/Generic.xaml#L853

LoRdPMN commented:
Reverted back and changed in https://github.com/ButchersBoy/MaterialDesignInXamlToolkit/blob/master/MaterialDesignThemes.Wpf/Themes/Generic.xaml#L853

ButchersBoy commented:
Thanks, welcome to contributors list!
